AUTHORS

```diff
@@ -63,7 +63,7 @@ Kevin C. Zimmerman <kevinczimmerman@gmail.com>
 Kirill Motkov <motkov.kirill@gmail.com>
 Konstantin Shaposhnikov <k.shaposhnikov@gmail.com>
 Leonid Kneller <recondite.matter@gmail.com>
-Lyron Winderbaum <lyron.winderbaum@student.adelaide.edu.au>
+Lyron Winderbaum <armadilloa16@gmail.com> <lyron.winderbaum@uwa.edu.au>
```

Comment:
Leave the Adelaide address here. It marks commits made by that user. Same in CONTRIBUTORS. If your commits are associated with the gmail address, you don't need to add the uwa address.

Reply:
Oooh ok, there isn't a way to say "don't try emailing me here"?

Reply:
Yes, they can email you there and get a bounce. This is really a contact list, it's a look-up table of authorships/contributors.

Reply:
Ok, makes sense, I'll just include all emails then.